Content.Server/Xenoarchaeology/Equipment/Systems/NodeScannerSystem.cs

```diff
@@ -14,10 +15,31 @@ public sealed class NodeScannerSystem : EntitySystem
     /// <inheritdoc/>
     public override void Initialize()
     {
-        SubscribeLocalEvent<NodeScannerComponent, AfterInteractEvent>(OnAfterInteract);
+        SubscribeLocalEvent<NodeScannerComponent, BeforeRangedInteractEvent>(OnAfterInteract);
```

Comment:
BeforeRangedInteractEvent is an attempt event which is used for denying an interact. It should not be used for doing interactions.

Reply:
Aren't attempt events something that inherit from `CancellableEntityEventArgs`?
`BeforeRangedInteractEvent` inherits from `HandledEntityArgs` and `EntityEventArgs`

Reply:
They are not structured that way, no.

Reply:
https://github.com/space-wizards/space-station-14/blob/18fa8bf633c73b7d0914194a4ddd5fed1cf93f0c/Content.Shared/Interaction/BeforeInteract.cs#L10

https://github.com/space-wizards/RobustToolbox/blob/40a90487043afa8eee4d9ae9d143aa6880bd14c6/Robust.Shared/GameObjects/EntityEvents.cs#L17

I may not be able to write good code, but I at least hope I can still understand whatever I'm reading, and that my IDE isn't lying to me

Reply:
Getting clarification. [Discord message link to #maintainers](https://discord.com/channels/310555209753690112/900426319433728030/1234366470985879664)

>Was BeforeRangedInteractEvent (Was named BeforeInteractEvent when created) supposed to be a cancellable attempt event? It was initially implemented to cancel interact events on VirtualItems, but it's now being used for Door Remote and Holosigns to do interactions.
>
>[BeforeInteract.cs BeforeRangedInteractEvent](https://github.com/space-wizards/space-station-14/blob/bc68ac96ddbf8cf1e44754e966e3fc7cb1d09d4a/Content.Shared/Interaction/BeforeInteract.cs)
>
>[Commit that created the file](https://github.com/space-wizards/space-station-14/commit/632e72b817c110dcda6fa008ff55d26136753be9)